frontend/src/AdminTeamListView.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+/**
+ * A component that displays a list of all the messages in our application.
+ */
+class AdminTeamListView extends React.Component {
+    constructor(props) {
+        super(props);
+
+        // This is a stateful component so we need to define a default representation of our state
+        this.state = {
+            text: '',
+            teams: null
+        };
+    }
+
+    render() {
+        if (this.state.teams == null) {
+            return (<div> loading teams</div>);
+        }
+        return (
+            <div>
+                <div>
+                    <textarea value={this.state.text} onChange={this.onTextChange.bind(this)}></textarea>
+                    <button onClick={this.onButtonClick.bind(this)} disabled={this.state.text.length === ''}> submit </button>
+                </div>
+                <div className="team-container">
+                    {this.state.teams.length === 0 && (<p> There are no teams yet </p>)}
+                    {this.state.teams.map((team) =>
+                        <div key={team.teamId} className="team-div">
+                            <div className="team-body">
+                                <li>
+                                    <Link to="/team/${team.cohortId}/${team.teamId}"> {team.teamName} </Link>
```

Comment:
You'll have to use the backticks otherwise it'll just render the text as is.
```suggestion
                                    <Link to=`/team/${team.cohortId}/${team.teamId}`> {team.teamName} </Link>
```